pyartifactory/models/user.py

```diff
@@ -1,10 +1,12 @@
 """
 Definition of all user related models.
 """
+from __future__ import annotations
 
 from datetime import datetime
 from typing import List, Optional
-from pydantic import BaseModel, EmailStr, SecretStr, HttpUrl
+
+from pydantic import BaseModel, EmailStr, HttpUrl, SecretStr
 
 
 class SimpleUser(BaseModel):

```

Comment:
I think there is missing "status" field as this model is used to deserialize list of users
https://jfrog.com/help/r/jfrog-rest-apis/get-user-list
also "uri" should be "pathlib.Path" at least or even better `Union[Path, str]` as @anancarv said

Reply:
@anancarv hm, my Artifactory instance queried on `GET /access/api/v2/users` only returns something like below in `uri` field: 
```
  }, {
    "username" : "johndoe",
    "realm" : "ldap",
    "status" : "enabled",
    "uri" : "/access/api/v2/users/johndoe"
  }, {
    "username" : "testuser",
    "realm" : "ldap",
    "status" : "enabled",
    "uri" : "/access/api/v2/users/testuser"
  } ]
  ```

Reply:
What's the correct type of `status` if it shall be added? `str`?

`uri` is unchanged then.

Reply:
Ok, well both fields `uri` and `status` should be changed to str then.

Reply:
From the jfrog documentation:

realm: An enum string with the possible values: internal, saml, oauth, ldap, crowd, scim, etc.
status: An enum string with the possible values: invited, enabled, disabled, locked
uri: A uri string to the Get Group Details endpoint

https://jfrog.com/help/r/jfrog-rest-apis/query-strings?tocId=LUOGfAviWOPYu8etpzPn6g

Reply:
ok it seems fine to leave it as string then